protocol/writing-resource/containment.feature

```diff
@@ -52,3 +52,56 @@ Feature: Creating a resource using PUT and PATCH must create intermediate contai
     When method GET
     Then status 200
     And match testContainer.parseMembers(response) contains intermediateContainer.url
+
+  Scenario: PUT conflicts when creating resource turning resource into container
+    * def requestUri = rootTestContainer.url + 'dahut'
+    Given url requestUri
+    And configure headers = clients.alice.getAuthHeaders('PUT', requestUri)
+    And header Content-Type = 'text/turtle'
+    And request '<> a <http://example.org/Dahut> .'
+    When method PUT
+    Then assert responseStatus >= 200 && responseStatus < 300
+
+    * def childrenRequestUri = rootTestContainer.url + 'dahut/foo/bar.txt'
```

Comment:
Minor point - grammatically, I would suggest `childRequestUri` since we are only dealing with one.
The `foo/` container is not actually necessary to this test is it, although it does no harm?